common/dictionaries.js

```diff
@@ -0,0 +1,9 @@
+export const HDPathsOrAlgosDictionary = {
```

Comment:
those should come from the API

Reply:
there is ed25519 missing for Polkadot

Reply:
I tried it, but it quite difficult to store these values in DB ([44, 118, 0, 0, 0]). At least I didn't find and easy way. Even storing an Array of strings was a headache :shrug: